2-0-data-structures-and-algorithms/src/test/java/com/bobocode/array_list/ArrayListTest.java

```diff
@@ -180,7 +218,12 @@ public void addElementByNegativeIndex() {
     @Test
     @Order(16)
     public void addElementByIndexLargerThanListSize() {
-        arrayList = ArrayList.of(4, 6, 11, 9);
+//        arrayList = ArrayList.of(4, 6, 11, 9);
+        try {
```

Comment:
@erneman1 avoid using `try-catch` in the test method, this can also be extracted to the helper method.